src/plugins/blocksmith/components/document-block-modal/index.jsx

```diff
@@ -109,19 +112,24 @@ class DocumentBlockModal extends React.Component {
   setBlock() {
     const {
       aside,
+      aspectRatio,
       hasUpdatedDate,
       launchText,
       selectedDocumentNode,
       selectedImageNode,
       updatedDate,
     } = this.state;
     const { block } = this.props;
-    return block.schema().createMessage()
-      .set('node_ref', NodeRef.fromNode(selectedDocumentNode))
+    const setBlock = block.schema().createMessage()
+      .set('aside', aside)
       .set('image_ref', selectedImageNode ? NodeRef.fromNode(selectedImageNode) : null)
       .set('launch_text', launchText || null)
-      .set('updated_date', hasUpdatedDate ? updatedDate : null)
-      .set('aside', aside);
+      .set('node_ref', NodeRef.fromNode(selectedDocumentNode))
+      .set('updated_date', hasUpdatedDate ? updatedDate : null);
+    if (aspectRatio !== AspectRatioEnum.AUTO) {
```

Comment:
@efjacobson - this doesn't seem right to me, IMHO we should save all the values user-selected (or just hydrated defaults for that matter) after clicking update. With the above `if` condition the `aspect_ratio` field of the block will never save the value `auto` while the UI shows the AUTO value in the `select` control. 

I would rather just set the initial state to just `block.get('aspect_ratio')` without setting any default value when it is `null` here: https://github.com/triniti/cms-js/pull/179/files#diff-47dfe088795b1ac392cdd6e9f1e96903R64

Reply:
@arunkarnati per Greg we are just going to set it to AUTO if they didnt touch the Select or if they did and selected auto